HW_3/templates/registration.html

```diff
@@ -0,0 +1,42 @@
+{% extends 'base.html' %}
+{% block title %}
+Registration
+{% endblock %}
+{% block content %}
+    <h1>Registration</h1>
+    <form method="POST" action="{{ url_for('registration') }}">
+        {{ form.csrf_token }}
+        {% with messages = get_flashed_messages(with_categories=true) %}
+            {% if messages %}
+                {% for category, message in messages %}
+                    <div>
+                        {{ message }}
+                    </div>
+                {% endfor %}
+            {% endif %}
+        {% endwith %}
+        <p>
+            {{ form.name.label }}<br>
+            {{ form.name(size=32) }}
+        </p>
+        <p>
+            {{ form.surname.label }}<br>
+            {{ form.surname(size=32) }}
+        </p>
+        <p>
+            {{ form.email.label }}<br>
+            {{ form.email(size=32) }}
+        </p>
+        <p>
+            {{ form.password.label }}<br>
+            {{ form.password(size=32) }}
+        </p>
+        <p>
+            {{ form.confirm_password.label }}<br>
+            {{ form.confirm_password(size=32) }}
+        </p>
+        <p>
+        <input type="submit" value="Registration">
+        </p>
```

Comment:
Или тут 
```suggestion
        {% for field in form if field.name != 'csrf_token' %}
	<p>
	<strong>field.label</strong>
	{{ field(size=32) }}
	</p>
	<ul class="alert alert-danger">
            {% for error in field.errors %}
            <li>{{ error }}</li>
            {% endfor %}
        </ul>
	{% endfor %}
	<p>
	{{ form.submit() }}
	</p>
```
Причем строки 
```html
        <ul class="alert alert-danger">
	    {% for error in field.errors %}
	    <li>{{ error }}</li>
	    {% endfor %}
	</ul>
```
повторяют сообщения flash об ошибках. Можно оставить что-то одно. Решать вам, что и где использовать.